### ISSUE TITLE
Is it possible to include coxphw in finalfit.coxph?

### ISSUE THREAD

**Issue by gustavomodelli** [ewenharrison/finalfit#66]
I am fitting a Cox model that does not meet hazard proportionality. In this case, it may be useful to use a weight cox regression. I find an excellent package coxphw. However, these package does not work well with tidy format. Is it possible to include coxphw in finalfit.coxph?
I appreciate some suggestions. I worked with Stratified cox models Stratified models but It will be no longer possible to make direct inferences on the effect associated with that variable.

**Comment by ewenharrison** [comment]
Many thanks for this. 
`coxphw` is not currently supported. 
It may be in the future, but it is not at the moment. 
An important question is to understand why the hazards are non-proportional. 
Depending on what you are doing, you may want to consider a more flexible parametric model using e.g. `flexsurv`.

**Comment by gustavomodelli** [comment]
Thank you very much. Congratulation on your excellent package finalfit. I have used it a lot in my statistical reports.

PS: Concerning non-proportional hazards, I am working with a 30 days mortality comparison between hemodialysis and transplant patients. In the first 10 days, the mortality was lower in transplant but after this period the mortality was higher (crossing survival curves). The HR was 1.4 in transplant patients but the proportionality hazards were violated. Then I used coxphw to estimate HR using weight cox regression.
![plot_survival](https://user-images.githubusercontent.com/32530756/116385106-ddca2680-a7ee-11eb-8fa5-2abbabe3e05e.jpg)
